test/evergreen.yml

```diff
@@ -1583,10 +1592,11 @@ buildvariants:
   run_on:
   - ubuntu1804-test
   expansions:
-    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd) CFLAGS="-fPIC -ggdb"
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
     posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-python --enable-zlib --enable-snappy --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
     make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    unit_test_args: -v 2
```

Comment:
Is this line necessary if the setting here is the same as the default value?

Reply:
Added this to make testing more robust and make it future proof. There are few other runtime arguments from unit test python script that we are not using. For example, we can make `long` unit test as default option very easily.

Reply:
It sounds like this is the preparation work for the `long` unit test, which is fine. I tend to do such restructure work in a more lazy way (at the time there's a real need), but all good if you choose to keep it.